### ISSUE TITLE
Minor issue: uncount() breaks if data argument is explicit

### ISSUE THREAD

**Issue by owasow** [elbersb/tidylog#43]
It appears `tidyr::uncount()` breaks if `data = ` argument is explicit. See example below:

```
library(tidyr)
library(tidylog)

library(conflicted)
for (f in getNamespaceExports("tidylog")) {
    conflicted::conflict_prefer(f, "tidylog", quiet = TRUE)
}

df <- tibble(x = c("a", "b"), n = c(1, 2))
uncount(df, n) # works fine
# uncount: now 3 rows and one column, ungrouped

uncount(data = df, weights = n)
# Error in log_summarize(.data, .fun = tidyr::uncount, .funname = "uncount",  : 
#   argument ".data" is missing, with no default
```

**Comment by elbersb** [comment]
Thanks, good catch!

All the function signatures in tidylog right now expect a ".data" argument, which comes from dplyr. When I expanded the package, we used ".data" for all other functions too, although the argument might have a different name. (I wonder if there are plans for standardizing that.) In other words, this problem does not only affect `uncount` but probably some other functions, too. I'll look into that, hopefully sometime soon.
